docs/production/requirements.md

```diff
@@ -113,6 +113,9 @@ on hardware requirements for larger organizations.
   private resources. If your network has its own outgoing HTTP proxy,
   Zulip supports using that instead.
 
+Zulip does not, itself, require SSH, but most installations will also require
+access to incoming port 22 for SSH access for remote access.
```

Comment:
Should this be a bullet?

Reply:
I was making it a non-bullet, because unlike everything else on the list, it's a non-requirement?  Though I guess the last two proxy bullets are somewhat similar on that axis.

Reply:
After looking at the generated build, I think I'm content with this as its own paragraph, so merging this.